docs/package_firewall/cargo.md

```diff
@@ -0,0 +1,116 @@
+# Cargo Package Firewall
+
+The Cargo package firewall is based on Cargo's sparse [Registry Index API].
```

Comment:
Should it be "Package Firewall" throughout our documentation? (a proper noun phrase)

Reply:
I think package firewall is not required to be a proper noun. In headlines it's nicer to read this way but it makes perfect sense even without knowing anything about it so I think it's fine the way it is.

This also is already present in the other ecosystem's docs so I think it should be fine.